src/triggers.js

```diff
@@ -240,6 +241,51 @@ function logTriggerErrorBeforeHook(triggerType, className, input, auth, error) {
   });
 }
 
+export function maybeRunAfterFindTrigger(triggerType, auth, className, results, config) {
+  return new Promise((resolve, reject) => {
+    const trigger = getTrigger(className, triggerType, config.applicationId);
+    if (!trigger) {
+      return resolve();
+    }
+    const request = getRequestObject(triggerType, auth, null, null, config);
+    const response = {
```

Comment:
The construction of that  hook doesn't seem to match the rest of the hooks.

the found objects should be available on `req.objects` not on `res.results`

Then the response object should have:

```
res.success(); // nothing, use the req.results
res.success(modifiedResults); // ignore req.results, and return modified results to the user
req.error(error); // Something wrong happened, return the error to the user
```

as well as the promise constructs as you handled them.

We should wait till req.success(), req.error() are called OR a returned promise has resolved.
If nothing is return from the afterFind function, we should wait till those methods are called. 
This is dangerous to introduce a new usage pattern as we expect to have something done in that trigger (unlike afterSave which doesn't have a req.success or req.error)